### ISSUE TITLE
currentPlaybackTime reported in Seconds and not MS on iOS

### ISSUE THREAD

**Issue by Kubessandra** [TheWidlarzGroup/react-native-video#3471]
# Bug

## Platform
Which player are you experiencing the problem on:
* iOS

## Environment info

Library version: 6.0.0-beta.3
Device: IPhone15

## Steps To Reproduce

1. Get an m3u8 stream and get the currentPlaybackTime

## Expected behaviour

1. Returning ms and not seconds

## Video sample
If possible, include a link to the video that has the problem that can be streamed or downloaded from.

(Making a PR to fix this)

**Comment by coofzilla** [comment]
I'm pretty sure thats a bug in this file [RCTVideo.swift](https://github.com/react-native-video/react-native-video/blob/11e5b756b14a2751352f97cde4fc027f441253ae/ios/Video/RCTVideo.swift#L285)

what it is:

```swift
"currentPlaybackTime": NSNumber(value: NSNumber(value: floor(currentPlaybackTime?.timeIntervalSince1970 ?? 0 * 1000)).int64Value),
```

what it should be:

```swift
"currentPlaybackTime": NSNumber(value: NSNumber(value: floor((currentPlaybackTime?.timeIntervalSince1970 ?? 0) * 1000)).int64Value)
```

**Comment by freeboub** [comment]
Will be available on 6.0.0-beta.4

**Comment by Kubessandra** [comment]
Thanks for the fast review and merge !
